src/wrap/mod.rs

```diff
@@ -42,33 +42,56 @@ impl Checker for Wrapper {
             SuggestionSet::new(),
             |mut acc, (origin, chunks)| {
                 for chunk in chunks {
-                    let mut new_lines = chunk
+                    let mut new_lines = dbg!(&chunk)
                         .as_str()
                         .split("\n\n")
                         .collect::<Vec<&str>>()
                         .iter()
-                        .map(|s| s.replace("\n", "") )
+                        .map(|s| s.replace("\n", ""))
```

Comment:
This won't work with common mark - needs to be context aware or dissect into pieces before hands.